dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/EventQueryParams.java

```diff
@@ -393,7 +393,7 @@ public String getKey() {
 
```

Comment:
[global note] About the base. We've finally settled on using 2.40.4.1 (or whatever branch points to the tag we are patching) instead of fix/ocg-40. This way, we can merge a PR to the fix branch without closing it. 

This also keeps the PRs isolated. I tried to change the base, but then I all the other PRs were included. @nshandra Can you do it? (also for other PRs, if any)